src/controllers/userController.js

```diff
@@ -1,15 +1,15 @@
-const { User } = require('../database/models')
-import  bcrypt from "bcrypt";
-import Jwt  from "jsonwebtoken";
+import bcrypt from "bcrypt";
+import Jwt from "jsonwebtoken";
+
+const { User } = require("../database/models");
 
 export class UserController {
   static async register(req, res) {
     try {
-    
       const { name, email, password } = req.body;
-      const userEmail = await User.findOne({ where: { email }});
+      const userEmail = await User.findOne({ where: { email } });
```

Comment:
Use a middleware to check this

Reply:
I did not work on this but because I rebased it reflected as a change I don't understand which middleware to use